packages/fish/build.sh

```diff
@@ -54,6 +55,8 @@ termux_step_post_make_install() {
 	function __fish_command_not_found_handler --on-event fish_command_not_found
 		$TERMUX_PREFIX/libexec/termux/command-not-found \$argv[1]
 	end
+
+	set -Ua fish_features no-keyboard-protocols
```

Comment:
- We should add a note that this is a temporary workaround until 
https://github.com/termux/termux-app/pull/4417
is landed in a release version of Termux.

Reply:
i've already mentioned that in the pr description. do you mean like a comment?

Reply:
Yes.